doc/api/http2.md

```diff
@@ -320,6 +320,17 @@ added: v9.4.0
 Will be `true` if this `Http2Session` instance has been closed, otherwise
 `false`.
 
+#### http2session.connecting
+<!-- YAML
+added: v9.12.0
```

Comment:
This should not be a fixed value. Instead of `v9.12.0` just put in: `REPLACEME`. This is going to be automatically be replaced during publishing a new release.

Reply:
Fixed in 8316d4f36b769fdf3224b65fdd32f4194a441ada